src/components/sbb-divider/readme.md

```diff
@@ -0,0 +1,23 @@
+# **name**
```

Comment:
Could you replace the placeholder and add a small documentation for this component?

Reply:
Done 👍